### DISCUSSION TITLE
Allow reading VCS ignore files without VCS config

### DISCUSSION THREAD

**Discussion by silversquirl** [biomejs/biome#391] [Ideas]
Specifically, it's nice to be able to use `.gitignore` without having a `.git` directory, eg. when building a Docker image you shouldn't need to copy the entire `.git` directory into the build container just to let Biome use the `.gitignore`

**Comment by ematipico** [comment]
That's a use case that never came up, and I'm glad you highlighted it. I believe it's a definitely valid suggestion and we should change the logic

**Comment by ematipico** [comment]
This was implemented in https://github.com/biomejs/biome/pull/435 and it will be available in the next release
